core/src/main/java/org/eclipse/hono/util/RegistryManagementConstants.java

```diff
@@ -271,6 +275,16 @@
      */
     public static final String FIELD_TRACING_SAMPLING_MODE_PER_AUTH_ID = "sampling-mode-per-auth-id";
 
+    /**
+     * The name of the JSON property containing the last modification date and time.
+     */
+    public static final String FIELD_UPDATED_ON = "updatedOn";
+
+    /**
+     * The name of the JSON property containing the version of the tenant or device or credentials information.
+     */
+    public static final String FIELD_VERSION = "version";
```

Comment:
are these fields used by the Device Registry Management API types? If not, then I believe they belong into the MongoDB module as they are implementation specific, aren't they?

Reply:
I have moved this to the utility class inside the MongoDB module.